test/ig/IGExporter.IG.test.ts

```diff
@@ -443,6 +443,30 @@ describe('IGExporter', () => {
       ]);
     });
 
+    it('should use the resolved version of a package when a dependency version is "latest"', () => {
+      config.dependencies = [{ packageId: 'hl7.fhir.us.core', version: 'latest' }];
+      exporter.export(tempOut);
+      const igPath = path.join(
+        tempOut,
+        'fsh-generated',
+        'resources',
+        'ImplementationGuide-sushi-test.json'
+      );
+      expect(fs.existsSync(igPath)).toBeTruthy();
+      const content = fs.readJSONSync(igPath);
+      const dependencies: ImplementationGuideDependsOn[] = content.dependsOn;
+      expect(loggerSpy.getAllLogs('error')).toHaveLength(0);
+      // Ensure US Core is exported with the resolved version
+      expect(dependencies).toEqual([
+        {
+          id: 'hl7_fhir_us_core',
+          uri: 'http://hl7.org/fhir/us/core/ImplementationGuide/hl7.fhir.us.core',
```

Comment:
I think this is an actively good change, but I just want to make sure since it wasn't explicitly mentioned in the issue or the PR. Because we now have a real version set, not just `"latest"`, the `uri` field is also (correctly, I think), taken from the IG, rather than the default we build [here](https://github.com/FHIR/sushi/pull/1409/files#diff-20b28aa731fc636ee110f2e0e3b7f52fe153d4a10cc1f78f8c9299463cda71f2R290).

So in this test, we get the US Core URI above, rather than the constructed `http://fhir.org/packages/hl7.fhir.us.core/ImplementationGuide/hl7.fhir.us.core`that released SUSHI would assign.

Reply:
Thank you for pointing this out! This is a side effect of the fact that changing the version means that the step that tries to get the URL from the IG now correctly finds the IG, since version is part of that search. I'm pretty sure there are no downsides to this.